src/inspector_profiler.cc

```diff
@@ -218,6 +224,47 @@ void V8CoverageConnection::End() {
   CHECK_EQ(ending_, false);
   ending_ = true;
   DispatchMessage("Profiler.takePreciseCoverage");
+  WriteSourceMapCache();
+}
+
+void V8CoverageConnection::WriteSourceMapCache() {
+  Isolate* isolate = env_->isolate();
+  Local<Context> context = env_->context();
+  HandleScope handle_scope(isolate);
+  Context::Scope context_scope(context);
+
+  Local<Function> source_map_cache_getter = env_->source_map_cache_getter();
+  Local<Value> source_map_cache_v;
+  Local<Value> argv[0] = {};
```

Comment:
The Windows build is failing because of this line:

> c:\workspace\node-compile-windows\src\inspector_profiler.cc(238): error C2466: cannot allocate an array of constant size 0 [c:\workspace\node-compile-windows\libnode.vcxproj]

You can omit allocating a variable here and use `nullptr` as the arguments list instead, that’s what we usually do in other places in the code.